.github/workflows/e2e-main.yml

```diff
@@ -143,8 +147,17 @@ jobs:
         jar xf target/uberjar/metabase.jar version.properties
         mv version.properties resources/
 
+    - name: Run OSS-specific Cypress tests
+      if: matrix.edition == 'oss'
+      run: |
+        yarn run test-cypress-run \
+        --env grepTags=@OSS \
+        --spec './frontend/test/metabase/scenarios/**/*.cy.spec.js'
+      env:
+        TERM: xterm
+
     - name: Run and record Cypress tests on ${{ matrix.folder }} - Master Branch
```

Comment:
Once we remove currents.dev from the project, this will get even more simplified.